op-e2e/actions/l2_runtime.go

```diff
@@ -304,6 +313,7 @@ func (rt *Runtime) depositToValPool(validator *L2Validator) {
 
 func (rt *Runtime) registerToValMgr(validator *L2Validator) {
 	minActivateAmount := rt.dp.DeployConfig.ValidatorManagerMinActivateAmount.ToInt()
+	minActivateAmount = new(big.Int).Mul(minActivateAmount, common.Big256)
```

Comment:
Do you have any specific reason for multiplying 256?

Reply:
Just big enough value. Since we added bond, `ValidatorManagerMinActivateAmount` is not enough to proceed all the tests.